<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="cardSelector">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cardreader</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rateSelector">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>48000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192000</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="gainSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>18</width>
      <height>211</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="gainDisplay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
